pkg/compiler/defer.go

```diff
@@ -3,6 +3,12 @@ package compiler
 import "go/ast"
 
 type deferInfo struct {
+	catchLabel   uint16
 	finallyLabel uint16
 	expr         *ast.CallExpr
 }
+
+const (
+	exceptionIndex        = 0
+	finallyProcessedIndex = 1
```

Comment:
These are like the best indexes (that have short load/store instructions), I don't think we want to spend them on exceptions.

Reply:
fixed.

Reply:
fixed.